application/config/routes.rb

```diff
@@ -19,8 +19,7 @@
 
     # post /projects/:project_id/uploads => create new collection
     # get /projects/:project_id/uploads => get collections from a project
-    resources :upload_collections, path: 'uploads', only: [ :create, :index ] do
-
+    resources :upload_collections, path: 'uploads', only: [ :create, :index, :edit, :update, :destroy ] do
```

Comment:
```suggestion
    resources :upload_collections, path: 'uploads', except: [ :show ] do
```

Reply:
I prefer naming the explicit set of actions